org.kde.spectacle.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "org.kde.spectacle",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-22.08",
+  "sdk": "org.kde.Sdk",
+  "command": "spectacle",
+  "rename-icon": "spectacle",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=org.kde.kglobalaccel",
+    "--talk-name=org.kde.plasmashell",
+    "--talk-name=org.kde.kwin"
```

Comment:
Add `--talk-name=org.freedesktop.FileManager1` to open folder from the app in file manager. 

Addresses @tsdgeos's concern